github/ci/prow/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```
@@ -0,0 +1,52 @@
+periodics:
+- name: periodic-kubevirtci-sync-crio-repository-weekly
+  interval: 168h
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+  spec:
+    containers:
+      - image: kubevirtci/bootstrap:v20201119-a5880e0
+        env:
+          - name: GOOGLE_APPLICATION_CREDENTIALS
+            value: /etc/gcs/service-account.json
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - >
```

New lines would be removed with `>`
```suggestion
          - |
```

---

Thanks, changed